docs/usage/self-hosted-configuration.md

```diff
@@ -219,6 +219,36 @@ For example:
 }
 ```
 
+## azureBypassPolicyReason
+
+By default the reason for bypassing policies is set to "Auto-merge by Renovate".
+
+Define a custom reason when policies are being bypassed to perform auto-merge.
+
+## azureBypassPolicyTypes
+
+Add a list of policy UUIDs which might be failing but should not block auto-merge.
+
+> For this to work the account that executes renovate must be allowed the `Bypass policies when completing pull requests` permission on the respective repository.
```

Comment:
Lets convert it to a !!!note so it's more visible and move

Reply:
Where do you want this to be moved?